src/utils/useLocationUpdateEffect.ts

```diff
@@ -33,6 +33,24 @@ const useLocationUpdateEffect = (
 
     return () => clearInterval(intervalId);
   }, [user, roomName]);
+
+  useEffect(() => {
```

Comment:
Currently counts multiple times if you have multiple tabs opened. Don't think it's a massive issue.

Reply:
add a comment explaining